project/dexter-monitor/routes/server.js

```diff
@@ -34,17 +34,39 @@ const _ = require('lodash');
 let serverList = [];
 let serverListLastModifiedTime = new Date();
 let checkingServerStatus;
+let doNotStartChecking;
 
-exports.init = function(doNotStartChecking){
+exports.init = function(_doNotStartChecking){
 
     mailing.init();
 
-    loadServerList()
+    doNotStartChecking = _doNotStartChecking;
+    updateServerListInternal();
```

Comment:
Why does it need to call updateServerListInteranal() when _doNotStartChecking is false or undefined?